research/long-context-llama/H2O/README.md

```diff
@@ -0,0 +1,11 @@
+## Run Llama with H2O for long context inference
+
```

Comment:
can you please add a link to the paper, blog post ( your name as the contributor/ anyone else you like to add) and also brief summary of

- what is H2O
- how it works
- what are advantage of it
- which models it supports (llama7, 13b 70B?) it seems we need to work with ecosystem projects like VLLM etc to get it done for multi-gpus?
- potential limitations

Reply:
Hi, @HamidShojanazeri. I will updated the PR by adding the implementation of Needle in a Haystack analysis, one example of inference with growing sequence length, as well as results on Llama-2/3. Please check the updated PR. Thanks!